README.md

```diff
@@ -7,20 +7,57 @@
 Free SVG icon font for popular brands. See them all on one page at <a href="https://simpleicons.org">SimpleIcons.org</a>. Contributions, corrections & requests can be made on GitHub. Started by <a href="https://twitter.com/bathtype">Dan Leech</a>.</p>
 </p>
 
-## Usage
+## Setup
+
+### From CDN
+
+```html
+<link rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/simple-icons-font@v2/font/simple-icons.min.css"
+      type="text/css">
+```
+
+### NodeJS
+
+```
+$ npm install simple-icons-font
+```
 
-### General Usage
+After installation, the icon font can then be found in `node_modules/simple-icons-font/font` along with stylesheets for a basic setup.
 
-This package can be installed into your project with NPM to provide all of our icons as an icon font.
+### Manual download
 
-```bash
-npm install simple-icons-font
+1. [Download the latest version of the package][npm-registry-tarball-link], extract it, and move the content of the `font` folder to your desired location.
+1. Reference the CSS file in a `link` tag of your HTML:
+
+```html
+<link rel="stylesheet"
+      href="/path/to/simple-icons.min.css"
+      type="text/css">
 ```
 
-After installation, the icon font can then be found in `node_modules/simple-icons-font/font` along with a stylesheet for
-a basic setup.
+## Usage
+
+The CSS files of simple-icons-font contains next classes:
+
+- `simpleicons`: Adds the font to the container that will display the icon.
+- `simpleicons--color`: Colorizes the icon with the main color of the brand.
+- `simpleicons-[ICON SLUG]`: Replace here `[ICON SLUG]` by with the correspondent slug for the icon. Their slugs correpond to filenames, which you can see listed in [simple-icons/icons][simple-icons--icons-dir-link].
+
+### Example
+
+```html
+<i class="simpleicons simpleicons--color simpleicons-simpleicons"></i>
+```
 
 ## Status
 
-[![Build Status](https://github.com/simple-icons/simple-icons-font/workflows/Verify/badge.svg)](https://github.com/simple-icons/simple-icons-font/actions?query=workflow%3AVerify+branch%3Adevelop)
-[![npm version](https://img.shields.io/npm/v/simple-icons-font.svg)](https://www.npmjs.com/package/simple-icons-font)
+[![Build Status][build-status-image]][build-status-link]
+[![NPM version][npm-version-image]][npm-package-link]
+
+[build-status-image]: https://img.shields.io/github/workflow/status/simple-icons/simple-icons-font/Verify/develop?logo=github
```

Comment:
I guess using this badge is a bit nicer, mind pushing an update for this to `develop` for the main repo as well? (Or open a PR in case you think other maintainers should have a look at this first)

Reply:
I think we should avoid direct commits as much as possible to appear as transparent as possible. I'll upload a pull so someone could learn something about how to use shields.io, since they use simple-icons for icons.